### PR TITLE
Add new fields to the showcase

### PR DESCRIPTION
**Pull request by kovalch**

### CODE REVIEW

**Review by sabinem — APPROVED** [review]

File: ckanext/switzerland/templates/showcase/snippets/showcase_info.html [hunk 1]
```diff
@@ -20,6 +20,17 @@ <h1 class="heading">{{ pkg.title or pkg.name }}</h1>
             {% if pkg.author %}
                 <span class="smallest">{{_('Submitted by')}}</span>
                 <p>{{ pkg.author }}</p>
+                {% if pkg.author_twitter %}
+                    <a href="{{ pkg.author_twitter}}" target="_blank">
```
Comment (bellisk) [inline ckanext/switzerland/templates/showcase/snippets/showcase_info.html:24]:
```suggestion
                    <a href="{{ pkg.author_twitter }}" target="_blank">
```

File: ckanext/switzerland/templates/showcase/snippets/showcase_info.html [hunk 2]
```diff
@@ -20,6 +20,17 @@ <h1 class="heading">{{ pkg.title or pkg.name }}</h1>
...
+                          <i class="fa fa-external-link"></i> {{pkg.author_twitter}}
+                    </a>
+                    <br>
+                {% endif %}
+                {% if pkg.author_github %}
+                    <a href="{{ pkg.author_github}}" target="_blank">
+                          <i class="fa fa-external-link "></i> {{pkg.author_github}}
```
Comment (bellisk) [inline ckanext/switzerland/templates/showcase/snippets/showcase_info.html:31]:
```suggestion
                    <a href="{{ pkg.author_github }}" target="_blank">
                          <i class="fa fa-external-link "></i> {{ pkg.author_github }}
```

Comment (kovalch) [reply ckanext/switzerland/templates/showcase/snippets/showcase_info.html]: fix in commit [884fbf8](https://github.com/opendata-swiss/ckanext-switzerland-ng/pull/223/commits/884fbf8a08fa28bfcfc66ca33d08f56e47ec2a06)

File: ckanext/switzerland/templates/showcase/snippets/showcase_info.html (hunk 1 above)
Comment (kovalch) [reply ckanext/switzerland/templates/showcase/snippets/showcase_info.html]: fix in commit [884fbf8](https://github.com/opendata-swiss/ckanext-switzerland-ng/pull/223/commits/884fbf8a08fa28bfcfc66ca33d08f56e47ec2a06)

File: ckanext/switzerland/templates/showcase/snippets/showcase_info.html [hunk 3]
```diff
@@ -20,6 +20,17 @@ <h1 class="heading">{{ pkg.title or pkg.name }}</h1>
             {% if pkg.author %}
                 <span class="smallest">{{_('Submitted by')}}</span>
                 <p>{{ pkg.author }}</p>
+                {% if pkg.author_twitter %}
+                    <a href="{{ pkg.author_twitter}}" target="_blank">
+                          <i class="fa fa-external-link"></i> {{pkg.author_twitter}}
```
[the remark this answers is not in the document]
Comment (kovalch) [reply ckanext/switzerland/templates/showcase/snippets/showcase_info.html]: fix in commit [884fbf8](https://github.com/opendata-swiss/ckanext-switzerland-ng/pull/223/commits/884fbf8a08fa28bfcfc66ca33d08f56e47ec2a06)

**Pull request merged**